lib/measured/unit_system_builder.rb

```diff
@@ -8,12 +8,51 @@ def unit(unit_name, aliases: [], value: nil)
     nil
   end
 
+  def si_unit(unit_name, aliases: [], value: nil)
+    @units += build_si_units(unit_name, aliases: aliases, value: value)
+    nil
+  end
+
   def build
     Measured::UnitSystem.new(@units)
   end
 
   private
 
+  def self.prefixes
```

Comment:
1. This actually isn't `private` (see [this](https://til.hashrocket.com/posts/ec20caa6ef-private-class-methods-in-ruby) link)
2. Let's just make this one a constant, `SI_PREFIXES`.